net/FlatBuffers/Table.cs

```diff
@@ -83,15 +84,15 @@ protected int __vector(int offset)
         }
 
         // Initialize any Table-derived type to point to the union at the given offset.
-        protected TTable __union<TTable>(TTable t, int offset) where TTable : Table
+        public T __union<T>(int offset) where T : struct, Accessor
```

Comment:
Hmm, you're right. Just looked over the 'Encode' code and it should all be in-place, low alloc stuff (string creation is the only one). I'll run the benchmark on another machine.